frontend/src/pages/AccountReceive.vue

```diff
@@ -37,14 +37,14 @@
               <div class="row justify-start q-col-gutter-md">
                 <base-input
```

Comment:
Now it will scan for all announcements when advanced mode is on and a private key is provided https://github.com/ScopeLift/umbra-protocol/commit/194834e49f178ff58dcb219493fc201df6e508bc

Reply:
Seems to work well, thanks!